res/values/strings.xml

```diff
@@ -607,6 +607,8 @@
     <string name="pref_send_copy_to_self">Send Copy to Self</string>
     <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat Folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
+    <string name="pref_only_fetch_mvbox_title">Only fetch from the DeltaChat folder</string>
+    <string name="pref_only_fetch_mvbox_explain">This will ignore all other folders. Regardless of this setting, the Spam folder is sometimes checked for replies to known messages.</string>
```

Comment:
I'm wondering if we should change this to

```suggestion
    <string name="pref_only_fetch_mvbox_explain">…and ignore all other folders. Regardless of this setting, the Spam folder is sometimes checked for replies to known messages.</string>
```

Reply:
i do not like the ellipsis-form - we do not use that somewhere else, it is overall not really used and looks a bit weird. also may make translations a bit harder as a sentence is split.

as the plain "Ignore all other folders." is even shorter, i'd vote for that :)